examples/tsx/pages/index.tsx

```
@@ -0,0 +1,13 @@
+import * as tsx from 'vue-tsx-support'
```

Please replace that by `import Vue from 'vue'` then use `Vue.extends({})` instead of `tsx.compoent({})` (which is not anymore in dependencies and should throw an error with current code).

Then we're good !

---

Oh...that is totally embarrassing 😑 

I pushed the fix!

---

Thanks for your time!